frontend/webadmin/modules/frontend/src/main/java/org/ovirt/engine/ui/frontend/server/dashboard/DashboardDataServlet.java

```diff
@@ -362,6 +369,20 @@ private GlobalUtilization lookupGlobalUtilization() throws DashboardDataExceptio
         return utilization;
     }
 
+    private String lookupEngineGrafanaBaseUrl(EngineLocalConfig config) {
+        try {
+            String grafanaFqdnVal = config.getEngineGrafanaFqdn();
+            String grafanaBaseUrlVal = config.getEngineGrafanaBaseUrl();
+
+            if (!grafanaFqdnVal.isEmpty() && !grafanaBaseUrlVal.isEmpty()) {
+                return grafanaBaseUrlVal;
+            }
```

Comment:
On branding we distinguish between 2 options and logged them both:
1. config file doesn't exist or empty (since Grafana is not installed)
2. config file exists but one of the properties within the file is invalid (no value).

Here you only log option 1. I guess this is ok since the branding will log in case option 2 occurs.

Reply:
Thinking about it a bit more, I don't think even the current check is needed since it is really a less robust copy of what is done in the `BrandingManager`.  In this servlet, I think it'll be just fine to look for the URL value and pass that back in the json response.  I'll update.